src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -1615,6 +1663,49 @@ const runtime::cpu::LayoutDescriptorPtrs&
     return result_layout_descriptors;
 }
 
+const vector<runtime::PerformanceCounter>& runtime::cpu::CPU_ExternalFunction::get_perf_counters()
+{
+    if (m_direct_execution)
+    {
+        return m_perf_counters;
+    }
+#if !defined(NGRAPH_DEX_ONLY)
```

Comment:
I don't understand why perf counters can't be used with dex.

Reply:
Not sure if I follow your concern. They are being added to DEX with this PR.